### PR TITLE
images: Fix systemd test

### PR DESCRIPTION
**Pull request by monstermunchkin**

### CODE REVIEW

**Comment by stgraber** [comment]
For the sysctl ones, it'd be better if we had the generator override the sysctl unit to ignore failures when applying the sysctls, I believe systemd-sysctl has a command line flag for that.

**Comment by monstermunchkin** [comment]
> For the sysctl ones, it'd be better if we had the generator override the sysctl unit to ignore failures when applying the sysctls, I believe systemd-sysctl has a command line flag for that.

The man page doesn't mention any such flag.

**Comment by monstermunchkin** [comment]
@stgraber I removed the sysctl modifications in favor of https://github.com/lxc/distrobuilder/pull/453

File: images/opensuse.yaml [hunk 1]
```diff
@@ -69,6 +69,13 @@ files:
 - path: /var/lib/dbus/machine-id
   generator: remove
 
+- path: /lib/modules-load.d/sg.conf
```
Comment (stgraber) [inline images/opensuse.yaml:72]: I don't like us removing files that come from packages as those are likely to show up again in a package update.
Can we maybe figure out what processes those and instead disable or ignore failures on the systemd unit instead?

Containers in general cannot load kernel modules, so disabling the logic entirely should be fine.

Comment (monstermunchkin) [reply images/opensuse.yaml:72]: Sure, that would be `systemd-modules-load.service`

**Pull request merged**